script/verification/SystemConfigUpgrade.s.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {Vm} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+import {ISystemConfig} from "./ISystemConfig.sol";
+import {IResourceMetering} from "./IResourceMetering.sol";
+import {console2 as console} from "forge-std/console2.sol";
+
+// SystemConfigUpgrade is a contract that can be used to verify that an upgrade of the SystemConfig contract
+// has not changed any of the storage variables that are not expected to change.
+// It stores the prior storage variables EXCLUDING:
+// - scalar
+// - basefeeScalar
+// - blobbasefeeScalar
```

Comment:
On reflection, this base contract seems less and less useful as a reusable component. So I might look to combine it with `SystemConfigUpgradeEcotoneScalars` and rename that to `SystemConfigUpgradeHolocene`.

Reply:
(It will still be reusable across L1 and L2 chain contexts, just not across upgrades to future versions).

Reply:
Done